src/index.js

```diff
@@ -161,6 +162,16 @@ domReady( () => {
 	// Modify any blocks we need to
 	addFilter( 'blocks.registerBlockType', 'blocks-everywhere/modify-blocks', modifyBlocks );
 
+	addFilter(
```

Comment:
This seems like it needs a check to only start filtering if the bbPress support is enabled.